src/status_im/contexts/profile/contact/header/view.cljs

```diff
@@ -0,0 +1,37 @@
+(ns status-im.contexts.profile.contact.header.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.theme]
+            [react-native.core :as rn]
+            [status-im.common.scalable-avatar.view :as avatar]
+            [status-im.contexts.profile.contact.header.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [scroll-y]}]
+  (let [{:keys [public-key customization-color
+                emoji-hash]
+         :as   profile}     (rf/sub [:contacts/current-contact])
+        customization-color (or customization-color :blue)
+        full-name           (profile.utils/displayed-name profile)
+        bio                 (:bio profile)
+        emoji-string        (string/join emoji-hash)
+        profile-picture     (profile.utils/photo profile)
+        online?             (rf/sub [:visibility-status-updates/online? public-key])
+        theme               (quo.theme/use-theme-value)]
+    [rn/view {:style style/header-container}
+     [rn/view {:style style/avatar-wrapper}
+      [avatar/view
+       {:scroll-y            scroll-y
+        :full-name           full-name
+        :online?             online?
+        :profile-picture     profile-picture
+        :border-color        (colors/theme-colors colors/white colors/neutral-95 theme)
+        :customization-color customization-color}]]
+     [quo/page-top
+      {:title            full-name
+       :description      :text
+       :description-text bio
+       :emoji-dash       emoji-string}]]))
```

Comment:
I think we can pass `emoji-hash` directly as `emoji-dash` to `quo/page-top`. I think that component will handle joining the emoji strings together.

Reply:
no it doesn't, we have to pass a string

Reply:
Can you please confirm that? It seems that the emoji-dash expects a sequence or vector of strings.
I'm looking at this code here: https://github.com/status-im/status-mobile/blob/bf1a734c4e99bdbb5d04bd85ba20d0921d839afd/src/quo/components/text_combinations/page_top/view.cljs#L109-L145

Reply:
Is the `emoji-hash` nil?

Reply:
If it's not a blocker for others, we don't have to worry about this now. I have a separate PR that adds the emoji-hash to the contact data, and I can handle stuff from there.

Reply:
at this moment for contact yes it's nil but also i checked in profile screen. if we pass emoji-hash (not nil) directly to page-top component it will throw an error

Reply:
Just for context, I think on that screen it uses a different page-top component that does require a string. And the one we use here is a different component that doesn't require a string.

Reply:
ah good point just let me check it

Reply:
yes, my bad i was checking with old component, in page-top we can directly pass emoji-hash and it's null safe. updated. thank you @seanstrom